docs/api_docs/maintenance/scale/horizontal-scale.md

```diff
@@ -0,0 +1,468 @@
+---
+title: Horizontal Scale
+description: How to scale a cluster, scale replicas, scale instances
+keywords: [horizontal scaling, Horizontal Scale]
+sidebar_position: 2
+sidebar_label: Horizontal Scale
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Horizontal Scale
+
+From v0.9.0, the horizontal scale can be divided into two types to better support various scenarios.
+
+- Final-state-oriented (operator: overwrite)
```

Comment:
The operator (overwrite/add/delete) in OpsRequest.spec.hscale has already been replaced by 'scaleIn' and 'scaleOut' fields, should work with @wangyelei to rewrite these descriptions.